github/pulls_reviews_test.go

```diff
@@ -195,6 +195,26 @@ func TestPullRequestsService_CreateReview_invalidOwner(t *testing.T) {
 	testURLParseError(t, err)
 }
 
+func TestPullRequestsService_UpdateReview(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/pulls/1/reviews/1", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "PATCH")
```

Comment:
s/PATCH/PUT/

Reply:
I am really sorry if I do not get this. Where should we put the `s/PATCH/PUT`?

Reply:
I apologize... that's shorthand for "substitute 'PATCH' with 'PUT'" (It's a vi/vim/sed [substitute command](https://vim.fandom.com/wiki/Search_and_replace)).
This is the same case as the comment above for `github/pulls_reviews.go`.